lldb/source/Commands/Options.td

```diff
@@ -805,19 +805,25 @@ let Command = "script add" in {
   def script_add_function : Option<"function", "f">, Group<1>,
     Arg<"PythonFunction">,
     Desc<"Name of the Python function to bind to this command name.">;
-  def script_add_class : Option<"class", "c">, Group<2>, Arg<"PythonClass">,
-  Desc<"Name of the Python class to bind to this command name.">;
+  def script_add_class : Option<"class", "c">, Groups<[2,3]>, 
```

Comment:
Let's swap the short-option between `script_add_class` & `script_add_completion_type`. Scripted Process uses `-C` (capital C) in `process launch` & `process attach` to specify the python class. Same for scripted thread plans with `thread step-scripted`.

Let's try to stay consistent for our users.

Reply:
I didn't add either of these options, they predate this patch.  This would break anybody that's adding scripted commands.  I don't remember why it's this way, but I don't think we can change it.